### ISSUE TITLE
Kansas (KS) Streamgage and reservoir data 

### ISSUE THREAD

**Issue by amabdallah** [WSWCWaterDataExchange/MappingStatesDataToWaDE2.0#274]
**Stream Gaging Network** seem federal
https://kwo.ks.gov/projects/stream-gaging-network

**Reservoir** seem federal
https://kwo.ks.gov/reservoirs

**Water Use**  time series data....not available
https://agriculture.ks.gov/divisions-programs/dwr/water-appropriation/water-use-reporting

**Comment by rwjam** [comment]
Agreed, these look to be federal sourced and sponsored data. Meaning they aren't state sourced and sponsored, so we wouldn't import them into WaDE.
- Looking through those links, it does look like that Stream Gaging Network data is through the [USGS](https://waterdata.usgs.gov/ks/nwis/rt)
- For that [reservoir](https://kwo.ks.gov/reservoirs) data, I'm not seeing any workable downloadable data to use, only PDFs and report.  Reading the description it seems like this is aimed at Bureau of Reclamation reservoirs operated in KS.
    - Looking at the [KS Data Portal](https://hub.kansasgis.org/), I didn't see anything about reservoirs pop out to me either.
    
Putting project on back log for now, due to missing state data to work with.
